magento/pwa-studio
# [bugfix]: check types in isRequired validate rule (#2302)

## Description
I added checking types for isRequired rule to detect true value also 
## Related Issue

Closes #2302 

### Verification Steps
Just add checkbox with isRequired rule
```html
<Checkbox field="agreement" label="check this" validate={isRequired} validateOnChange/>
```
